include/inexor/vulkan-renderer/world/cube.hpp

```diff
@@ -161,6 +170,17 @@ class Cube : public std::enable_shared_from_this<Cube> {
     /// Recursive way to collect all the caches.
     /// @param update_invalid If true it will update invalid polygon caches.
     [[nodiscard]] std::vector<PolygonCache> polygons(bool update_invalid = false) const;
+
+    /// Get the (face) neighbor of this cube by using a similar implementation to Samets "OT_GTEQ_FACE_NEIGHBOR(P,I)".
+    /// @brief Get the (face) neighbor of this cube.
+    /// @param axis The axis on which to get the neighboring cube
+    /// @param direction Whether to get the cube which is above or below this cube on the selected axis
+    /// @returns Same-sized neighbor if existent, else larger neighbor if exists, otherwise (i.e. when no neighbor
+    /// exists) nullptr.
+    /// @see Samet, H. (1989) [Neighbor finding in Images Represented by Octrees.]
+    /// (http://web.archive.org/web/20190712063957/http://www.cs.umd.edu/~hjs/pubs/SameCVGIP89.pdf)
+    /// Computer Vision, Graphics, and Image Processing. 46 (3), 367-386.
```

Comment:
```suggestion
    /// @see Samet, H. (1989) <a href="https://web.archive.org/web/20190712063957/http://www.cs.umd.edu/~hjs/pubs/SameCVGIP89.pdf">
    /// Neighbor finding in Images Represented by Octrees</a>.
    /// Computer Vision, Graphics, and Image Processing. 46 (3), 367-386.
```

At least link to the https version of web.archive.org

Reply:
You mentioned on discord, that you wanted to use the html tag instead of markdown because the markdown didn't generate a link.
However, for me it did. So i suggest keeping the markdown and if readthedocs works with it, then we'll keep it that way. Otherwise I'll create a PR with the html tag. Is that okay for you?